cmd/dagger/cmd/lsp.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var lspCmd = &cobra.Command{
-	Use:    "lsp",
+	Use:    "cuelsp",
 	Short:  "Run Dagger CUE Language Server",
```

Comment:
Should we document that we need to press 3 times CTRL+C to stop it?
It's not classical to do that, so I guess we should document it.
Or at least log it when the first one is pressed that we're shutting down nicely or that we need 2 more to stop the server.

We could put it in the long help description of it.

Reply:
Noted, thanks. Not a blocker, you're right